Sources/SKCore/PathPrefixMapping.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+public struct PathPrefixMapping {
```

Comment:
I could just use `PathMapping` from IndexStoreDB instead and it import it/depend upon it from SKCore, LMK which you'd prefer.

Reply:
I don’t really have a strong opinion. If this struct stays as simple as it its, avoiding the dependency sounds good to me 👍